release/pkg/manager/calico/manager.go

```diff
@@ -358,13 +358,13 @@ func (r *CalicoManager) getRegistryFromManifests() (string, error) {
 	}
 	imgs := strings.Split(out, "\n")
 	for _, i := range imgs {
-		if strings.Contains(i, "operator") {
-			continue
-		} else if strings.Contains(i, "tigera/") {
-			splits := strings.SplitAfter(i, "/tigera/")
-			registry := splits[0]
-			logrus.WithField("registry", registry).Debugf("Using registry from image %s", i)
-			return registry, nil
+		if strings.HasPrefix(i, "calico/") {
```

Comment:
I don't quite understand this change - what does it do? We're ignoring "calico/" images? 

The old implementation had a log that emitted what registry was chosen and why - I think we should at least keep that.

Reply:
it is not ignoring calico images. The images are currently listed as `calico/<image_name>` so when the image is set as `calico/<image_name>`, this passes an empty string as a result.

Reply:
I don't understand why we would want to do that though. 

Maybe you could explain what was wrong with the previous implementation and that will help me understand the change.